corehq/apps/userreports/operators.py

```diff
@@ -6,6 +6,7 @@
 NOT_EQUAL = lambda input, reference: input != reference
 IN = lambda input, reference_list: input in reference_list
 IN_MULTISELECT = lambda input, reference: reference in (input or '').split(' ')
+ANY_IN_MULTISELECT = lambda input, reference: any([subval in (input or '').split(' ') for subval in reference])
```

Comment:
@czue 
Yes reports use this.

Operator which I added works: For example in xforms we have property ```test = "1 2 3 4 5"``` and in reports we have calculator with condition (if any value from ```test``` in ```[2, 3]```), then we check if any of values from ```test``` property are in the list ```[2, 3]```, but ```in_mulit``` we only can check one value.

Is that make sanse?